Packs/RiskIQDigitalFootprint/pack_metadata.json

```
@@ -0,0 +1,51 @@
+{
+    "name": "RiskIQ Digital Footprint",
+    "description": "RiskIQ Digital Footprint integration enables your security team to manage assets outside your firewall, by bringing its details, providing you to add or update assets and analyze your digital footprint from the view of the global adversary with the help of various commands that summarise the activities performed on your assets.",
```

```suggestion
    "description": "RiskIQ Digital Footprint integration enables your security team to manage assets outside your firewall and provides you with the ability to add or update assets and analyze your digital footprint from the view of the global adversary.",
```
@crestdatasystems the language here was a bit unwieldy. Are you ok with this updated description?

---

@avidan-H Yes, we are ok with this description.